composer/utils/eval_client/eval_client.py

```diff
@@ -12,7 +12,7 @@
 
 
 class EvalClient(abc.ABC):
```

Comment:
should just delete the base class. the deprecations of the subclasses was intended to indicate deprecation of the whole concept, without adding a constructor to the abstract base class

Reply:
sure can do